poolqc/poolingQC_sample.py

```diff
@@ -0,0 +1,300 @@
+import dash_core_components as dcc
+import dash_html_components as html
+import dash.dependencies as dep
+import dash_table as dt
+import pandas
+import plotly.plotly as plotly
+import plotly.graph_objs as go
+import numpy as np
+
+pandas.options.mode.chained_assignment = None
+index = pandas.read_hdf('/home/ijafari/cache_files/rnaseqqc_cache.hd5')
+
+all_runs = index['Sequencer Run Name'].sort_values(ascending=False).unique()
+all_runs = [x for x in all_runs if str(x) != 'nan']
```

Comment:
Can `dropna` or `notnull` be used in place of `!= 'nan'`?

Reply:
all_runs is a list and df.dropna or df.notnull doesn't seem to work on it

Reply:
Ah, it looks like the sequencer run name column already contains the string literal `'nan'`. The cast to `str(x)` is not strictly necessary, but probably worth keeping in to future-proof it.